gemrb/core/DisplayMessage.cpp

```diff
@@ -37,6 +37,8 @@ static const auto DisplayFormatValue = FMT_STRING(L"[p][color={:08X}]{}: {}[/col
 static const auto DisplayFormatNameString = FMT_STRING(L"[color={:08X}]{} - [/color][p][color={:08X}]{}: {}[/color][/p]");
 static const auto DisplayFormatSimple = FMT_STRING(L"[p]{}[/p]");
 
+const char* GUIColors[] = { "DIALOG", "DIALOG_PARTY", "GOT_GOLD", "RED", "GOT_XP", "LIGHT_GREY", "WHITE" };
```

Comment:
I suggest you convert this to an empty vector that you then fill up in 
`DisplayMessage::DisplayMessage()` by reading the color table and storing the rownames.